speakeasy/winenv/api/kernelmode/ntoskrnl.py

```diff
@@ -181,6 +187,10 @@ def _vsnprintf(self, emu, argv, ctx={}):
 
         return rv
 
+    @apihook('vsprintf_s', argc=4, conv=_arch.CALL_CONV_CDECL)
+    def vsprintf_s(self, emu, argv, ctx={}):
+        return self._vsnprintf(emu, argv, ctx)
```

Comment:
Probably not 100% correct because it should crash when the buffer is too small, but that shouldn't matter for the happy path.